### ISSUE TITLE
silentMoveTo still shows animation when scrolling to a slide

### ISSUE THREAD

**Issue by adriennetacke** [alvarotrigo/fullPage.js#1835]
Hello! 

Please check out the quick fiddle: 
http://jsfiddle.net/zegm4xag/1/

Here, it still shows the animation instead of cutting straight to the requested slide. 

Is there something I am missing?

**Comment by alvarotrigo** [comment]
Not missing anything. That's a bug.

**Comment by adriennetacke** [comment]
Ah, ok. Thanks for validating!

**Comment by alvarotrigo** [comment]
Do this meanwhile:

```javascript
$('.move').click(function() {
	$.fn.fullpage.setScrollingSpeed(0);
    $.fn.fullpage.moveTo(0, 3);
    $.fn.fullpage.setScrollingSpeed(700);
});
```

**Comment by adriennetacke** [comment]
Perfect. Thank you.

**Comment by nhylated** [comment]
Hi @alvarotrigo. I've fixed this in my dev branch [here](https://github.com/nhylated/fullPage.js/commit/f715d9c49c433f12199f82e056276fff95765b99). Can I create a pull request?

**Comment by alvarotrigo** [comment]
@nhylated yep, you may.

**Comment by nhylated** [comment]
@alvarotrigo Done - #1885 

Thanks!

**Comment by alvarotrigo** [comment]
Fixed on the `dev` branch 2.7.9

**Comment by alvarotrigo** [comment]
Fixed in 2.7.9
